overlord/ifacestate/helpers.go

```diff
@@ -95,6 +98,62 @@ func (m *InterfaceManager) addSnaps() error {
 	return nil
 }
 
+func (m *InterfaceManager) unbreakTheWorld() error {
+	// XXX: As a special measure to unbreak the world refresh seccomp security
+	// of all the snaps on the system. Later on this should be improved to
+	// either refresh all security backends or to know how to apply stashed
+	// (versioned) security that was preserved by snapd on upgrade to a new
+	// version of itself.
+
+	// Get all the security backends
+	securityBackends := m.repo.Backends()
+
+	// Get all the snap infos
+	snaps, err := snapstate.ActiveInfos(m.state)
```

Comment:
this gives you only active snaps (you need All to get all of them),   I know we recently changed enable/disable to remove profiles os inactive ones don't have them, but not sure if it means there is something to do for old inactive stuff